design/index.md

```diff
@@ -40,4 +40,11 @@ Other tools included in the Creative Cloud suite that may be useful include:
 
 ### Blender ([more info](https://blender.org))
 
-Blender is a free and open-source 3D computer graphics software toolset. It can be used to create static renders of 3d assets, or entire 2d & 3d animations. 
\ No newline at end of file
+Blender is a free and open-source 3D computer graphics software toolset. It can be used to create static renders of 3d assets, or entire 2d & 3d animations. 
+
+### Websites
```

Comment:
When referring to "websites" - is this just our own FlowForge website, or encompassing the app.flowforge.com too?

Reply:
I don't think this should be used for the app, I think the best practices will not be the same. What do you think?

Reply:
I agree - just wanted to clarify